### PR TITLE
Feature/create config

### LINKED ISSUE #61: Create the configuration creation flow
Create the flow that enables users to create a new configuration. This configuration needs to be stored on disk as well. (see #60) 

Styling the flow is not inside scope of this issue.

### PR DESCRIPTION
**Pull request by FreekDS**
Closes #61

### CODE REVIEW

File: src/interfaces/Configuration.tsx [hunk 1]
```diff
@@ -1,4 +1,14 @@
-interface Configuration {
+export interface Policy {
+    lhs: string;
```
Comment (BramDC3) [inline src/interfaces/Configuration.tsx]: I think it's a good idea to not use acronyms as variable names. Can you rename lhs and rhs? :)

Comment (FreekDS) [reply src/interfaces/Configuration.tsx]: sure will do

File: src/renderer/components/ConfigurationTable/ConfigurationTable.tsx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+import EditIcon from 'Assets/icons/icons8-edit-property-500.svg';
+import TrashIcon from 'Assets/icons/icons8-delete-bin-500.svg';
+
+
+interface ConfigTableProps {
+    configs: Array<Configuration>;
+    selectable?: false;
+    currentSelected?: null;
+    setCurrentSelected?: null;
+
```
Comment (BramDC3) [inline src/renderer/components/ConfigurationTable/ConfigurationTable.tsx]: blank line for no reason :o

File: src/renderer/components/ConfigurationTable/ConfigurationTable.tsx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+            { name: "IccProfileClass", operator: '=', value: 'input' },
+            { name: "ImageWidth", operator: '>', value: 500 },
+            { name: "ImageHeight", operator: '<=', value: 300 }
+        ], reports: ["JSON", "PDF"]
+    },
+    { name: "Extended", implementation: "Baseline TIFF 6.0", policies: [], reports: ["JSON", "PDF"] },
+    { name: "Extended", implementation: "Baseline TIFF 6.0", policies: [], reports: ["JSON", "PDF"] },
+    { name: "Extended", implementation: "Baseline TIFF 6.0", policies: [], reports: ["JSON", "PDF"] },
+    { name: "Extended", implementation: "Baseline TIFF 6.0", policies: [], reports: ["JSON", "PDF"] },
+
```
Comment (BramDC3) [inline src/renderer/components/ConfigurationTable/ConfigurationTable.tsx]: Same here

File: src/renderer/components/CreateConfigStages/AddPolicy/AddPolicy.tsx [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+    /**
+     * Get the allowed operators of a certain policy rule
+     * @param policyRule policy rule to get the allowed operators from
+     */
+    const getAllowedOperators = (policyRule: PolicyRule) => {
+        const ALL = ["<", ">", "="];
+
+        if (policyRule.allowAllOperators) {
+            return ALL;
+        }
```
Comment (BramDC3) [inline src/renderer/components/CreateConfigStages/AddPolicy/AddPolicy.tsx:94]: There needs to be a blank line between the if and the switch

File: src/renderer/components/FileUploadStages/PickConfiguration/Stage2.tsx [hunk 5]
```diff
@@ -115,28 +85,8 @@ const useStyles = makeStyles((theme: Theme) =>
 );
 
 
-// Styled Material UI Table Cell component
-const TableCell = withStyles({
-    root: {
-        borderBottom: "none"
-    }
-})(MuiTableCell);
-
```
Comment (AmberPerard) [inline src/renderer/components/FileUploadStages/PickConfiguration/Stage2.tsx:124]: blank lines?

**Review by BramDC3 — APPROVED** [review]

File: src/renderer/components/FileUploadStages/PickConfiguration/Stage2.tsx (hunk 5 above)
Comment (FreekDS) [reply src/renderer/components/FileUploadStages/PickConfiguration/Stage2.tsx:124]: That piece of code does not exist anymore in this pull request, thats wy there is a '-' sign in front of the line 🙂

**Pull request merged**